src/modules/git/calculations.js

```diff
@@ -0,0 +1,29 @@
+export let fileMap = {};
+export let finalcount = 0;
+
+export default function doTheCalculations(newFileMap) {
+  finalcount = newFileMap.length;
+  fileMap = {};
+
+  for (let i = 0; i < finalcount; i += 1) {
+    const { file } = newFileMap[i];
+    if (!fileMap[file]) {
+      fileMap[file] = {
+        file,
+        additions: +newFileMap[i].stats[0],
```

Comment:
Altough its a correct way to cast the string to a number, its very hard to read. I would prefer, if you use parseInt(xy). But thats just personal.  But as you are doing it a lot, i think it would be the best, to prepare the data in an extra block or function. You could use this one-liner (should work - didnt try it out). 
```javascript
newFileMap[i].stats= newFileMap[i].stats.map(item=>parseInt(item));
```